Makefile

```diff
@@ -12,7 +12,6 @@ endif
 test: clean
 	./manage.py test --settings=$(test_settings) --with-coverage --with-ignore-docstrings \
 		--exclude-dir=notesserver/settings --cover-inclusive --cover-branches \
-		--ignore-files=search_indexes.py --ignore-files=highlight.py\
```

Comment:
Is this not fixed in a newer version? If not, can we comment out the line and add an explanation instead?